src/com/twitter/intellij/pants/completion/PantsCompletionContributor.java

```diff
@@ -55,15 +63,28 @@ public PantsCompletionContributor() {
     protected void addCompletions(
       @NotNull CompletionParameters parameters, ProcessingContext context, @NotNull CompletionResultSet set
     ) {
+
       final PsiFile psiFile = parameters.getOriginalFile();
       if (!PantsUtil.isBUILDFileName(psiFile.getName())) {
         return;
       }
-      Collection<String> addresses = PantsBuildFileIndex.getFiles(psiFile);
-      for (String address : addresses) {
-        set.addElement(LookupElementBuilder.create(address));
+
+      if (isDependenciesString(parameters)) {
+        Collection<String> addresses = PantsBuildFileIndex.getFiles(psiFile);
```

Comment:
This works to some extent, but the real target addresses are stored in the modules, you may be get it from 
https://github.com/pantsbuild/intellij-pants-plugin/blob/58d0b404cb2cadb3b87f11770715323f9b866cbd/common/com/twitter/intellij/pants/util/PantsUtil.java#L455-L467

so they should appear in the format of:
`<path/to/target>:<name>`
instead of just `<path/to/target>`